sample/src/js/connectivity-ui.js

```diff
@@ -144,15 +144,15 @@ export function graphIntermediateStats(mediaType, stats) {
   if (!charts[mediaType]) {
     charts[mediaType] = createChart(mediaType);
   }
-  const bitsReceived = mediaStats && mediaStats.bytesReceived ? mediaStats.bytesReceived * 8 : 0;
+  const bitsSent = mediaStats && mediaStats.bytesSent ? mediaStats.bytesSent * 8 : 0;
   resultCount[mediaType]++;
   charts[mediaType].series[0].addPoint({
     x: resultCount[mediaType],
-    y: bitsReceived - prevBitsReceived[mediaType]
+    y: bitsSent - prevBitsReceived[mediaType]
   }, true, false);
   const chartTitle = (stats.phase === 'audio-only') && (mediaType === 'video') ?
    'Testing audio-only stream' :
    'Bitrate over ' + resultCount[mediaType] + 'sec';
   charts[mediaType].setTitle(null, { text: chartTitle});
-  prevBitsReceived[mediaType] = bitsReceived;
+  prevBitsReceived[mediaType] = bitsSent;
```

Comment:
When I run the sample, there are two charts: audio bitrate stability and video bitrate stability. Both yield flat results with a constant 0kBps over 29 seconds. This seems broken.

Reply:
Please can you double-check again? I re-test that from scratch and it works fine on my side.

Reply:
Sorry. Issues on my side.